app/src/main/java/org/p2p/wallet/restore/ui/keys/SecretKeyFragment.kt

```diff
@@ -1,50 +1,28 @@
 package org.p2p.wallet.restore.ui.keys
 
+import androidx.core.content.FileProvider
 import android.content.ActivityNotFoundException
 import android.content.Intent
 import android.os.Bundle
-import android.text.SpannableString
-import android.text.Spanned
-import android.text.TextPaint
-import android.text.method.LinkMovementMethod
-import android.text.style.ClickableSpan
 import android.view.View
-import android.widget.LinearLayout
-import androidx.annotation.StringRes
-import androidx.core.content.FileProvider
-import androidx.core.view.children
-import androidx.core.view.isVisible
-import com.google.android.flexbox.FlexDirection
-import com.google.android.flexbox.FlexboxLayoutManager
-import com.google.android.flexbox.JustifyContent
 import org.koin.android.ext.android.inject
-import org.p2p.uikit.utils.attachAdapter
+import org.p2p.uikit.organisms.seedphrase.SecretKey
 import org.p2p.uikit.utils.hideKeyboard
-import org.p2p.uikit.utils.showSoftKeyboard
 import org.p2p.uikit.utils.toast
 import org.p2p.wallet.BuildConfig
 import org.p2p.wallet.R
 import org.p2p.wallet.common.analytics.constants.ScreenNames
 import org.p2p.wallet.common.analytics.interactor.ScreensAnalyticsInteractor
 import org.p2p.wallet.common.mvp.BaseMvpFragment
-import org.p2p.wallet.common.ui.widget.ProgressButton
 import org.p2p.wallet.databinding.FragmentSecretKeyBinding
-import org.p2p.wallet.restore.model.SecretKey
 import org.p2p.wallet.restore.ui.derivable.DerivableAccountsFragment
-import org.p2p.wallet.restore.ui.keys.adapter.SecretPhraseAdapter
-import org.p2p.wallet.settings.ui.reset.seedinfo.SeedInfoFragment
-import org.p2p.wallet.utils.getDrawableCompat
+import org.p2p.wallet.utils.getClipboardText
 import org.p2p.wallet.utils.popBackStack
 import org.p2p.wallet.utils.replaceFragment
-import org.p2p.wallet.utils.unsafeLazy
-import org.p2p.wallet.utils.viewbinding.context
 import org.p2p.wallet.utils.viewbinding.viewBinding
 import timber.log.Timber
 import java.io.File
 
-private const val SEED_PHRASE_SIZE_SHORT = 12
-private const val SEED_PHRASE_SIZE_LONG = 24
-
 class SecretKeyFragment :
```

Comment:
Could you rename this fragment before merging? 
SeedPhraseFragment would be better, whatcha think?

Reply:
Yes, okay